### ISSUE TITLE
Show an user friendly message when a secret is not found

### ISSUE THREAD

**Issue by rberrelleza** [okteto/okteto#2033]
**Describe the bug**
We should show a user-friendly secret when the file mentioned in okteto.yml is not avaiable.

**To Reproduce**

1. Add a`secret` to your okteto.yaml , with a file that doesn't exist
2. Run` okteto up` 

- Version 1.14.8

**Additional context**
Add any other context about the problem here (e.g. okteto.yml, k8s manifests, logs)

![image](https://user-images.githubusercontent.com/475313/143962643-ed6af5f2-f99b-4709-a96d-392e7df95836.png)

**Comment by RinkiyaKeDad** [comment]
Using the code in the current `master` branch, the error message which gets shown is:

![image](https://user-images.githubusercontent.com/56963264/145786243-740038a9-9134-449f-ae90-d4412d17bcb2.png)

This is the standard error message returned on calling `yaml.UnmarshalStrict` [here](https://github.com/okteto/okteto/blob/1f927579fa/pkg/model/dev.go#L325) when it's not able to find any of the files specified in the YAML. I'm not sure how we can go about identifying that it is one of the `secrets` which is being supplied a non-existing file, any ideas?

**Comment by pchico83** [comment]
@jLopezbarb are we generating that error in our own serialization logic?

**Comment by jLopezbarb** [comment]
Yes, it's generated here: https://github.com/okteto/okteto/blob/master/pkg/model/serializer.go#L389
